### ISSUE TITLE
wilcity service client not working

### ISSUE THREAD

**Issue by byrehep** [wilokecom/wilcityservice-client#1]
wilcity service client not working? I have installed version 1.1.8. But it doesn't work! How can I fix ?

<a href="https://ibb.co/kX4gXH1"><img src="https://i.ibb.co/qpNMpy5/sorun1.png" alt="sorun1" border="0"></a>

<a href="https://ibb.co/jrKq26f"><img src="https://i.ibb.co/2k2RQnh/sorun2.png" alt="sorun2" border="0"></a>

**Comment by renrax** [comment]
Yes. It doesn't work for me either. Plugins are not displayed. I deleted everything and put it all over again. 
Oops! We could not find any plugin
Oops! We could not find this theme.
Although everything is active and the token is registered.

**Comment by zweigelb** [comment]
Installed the latest version of the theme and the latest version of the service client 1.1.8
Secret token is set but we can't update the plugins anymore.
